templates/.circleci/config.yml

```diff
@@ -78,46 +90,33 @@ jobs:
       - restore_cache:
           key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
-          name: NPM Rebuild
-          command: npm install
+          name: Install Latest NPM.
+          command: npm i -g npm@latest
+      - run:
+          name: NPM Install.
+          command: npm ci
       - run:
           name: Run linting.
           command: npm run lint
       - run:
-          name: Run NSP Security Check.
+          name: Run NPM Audit.
           command: npm run security
       - run:
-          name: Validate Commit Messages
+          name: Validate Commit Messages.
```

Comment:
I think we shouldn't test commit messages with the CI Setup and remove this analog to `publish`

Reply:
@michael-ciniawsky You yourself argued that you must always write correctly commit message to easy release :confused: We will not be able to always monitor the messages in all the repositories. It is better to leave it.

Reply:
Reviewers (for PR's) and the person doing a release needs to validate/verify the commits and checking the `CHANGELOG`. Currently 

```
'ci:lint:commits': 'commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}'
```

would likely constantly fail the CI if one does the following while developing 

```
fix(file): message
squash later ❌ 
not relevant  ❌ 
fix ❌ 
fix ❌ 
refactor ❌ 
```

not to mention that many people don't use Conventional Commits at all

Reply:
@michael-ciniawsky it is normal what you should rewrite commit message if you `squash` commits, also it is protect if you accidentally wrote a new commit message with a typo (master will be failed and you should rewrite commit message). A lot of repos using this check. I don't think we should remove this. And we other developers should use right commit message after PR. 

> not to mention that many people don't use Conventional Commits at all

But we use this and it is normal what we lint this.

Reply:
Sure, but the point is that a failing Ci is too strict here imho. I normally ensure and update the commit message for most PR manually anyways. If it only lints the first commit then maybe but for all commits it will definitely cause more problems then it solves and we don't care about those commits in most cases

Reply:
@michael-ciniawsky feel free to open issue, if you still thinks it is unnecessary, out of scope this pr